src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -168,6 +168,51 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="PackageReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="PackageReference" _locComment="" -->Reference to a package
```

Comment:
Apparently they get localized similar to the xaml files, though it's not quite clear to me what the final destination of the xsd files is in the VS build layout. Added a note for next sprint